### PR TITLE
Create ingress-with-annotations.yaml

### PR DESCRIPTION
**Pull request by prageethw**
sample to enable ingress with some annotations.

### CODE REVIEW

File: deploy/examples/ingress-with-annotations.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger
```
Comment (jpkrohling) [inline deploy/examples/ingress-with-annotations.yaml:4]: We typically use the example name in this field: `name: ingress-with-annotations` would be more appropriate.

Still on the topic of naming: I think the example name (and file name) should actually be: `ingress-with-host`.

File: deploy/examples/ingress-with-annotations.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+spec:
+  strategy: allInOne 
+  storage:
+    type: memory 
+    options:
```
Comment (jpkrohling) [inline deploy/examples/ingress-with-annotations.yaml:9]: Could you please leave the example with only the minimum set of fields required to work?

File: deploy/examples/ingress-with-annotations.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  strategy: allInOne 
+  storage:
+    type: memory 
+    options: 
+      memory: 
+        max-traces: 1000000
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: nginx
```
Comment (jpkrohling) [inline deploy/examples/ingress-with-annotations.yaml:15]: Is this needed?

File: deploy/examples/ingress-with-annotations.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    hosts:
+      - mesh-jaeger.xxx.com #your domain name.
+  agent:
```
Comment (jpkrohling) [inline deploy/examples/ingress-with-annotations.yaml:18]: This node can also be removed.

**Review by jpkrohling — COMMENTED** [review]
Thank you for providing this example! I made a few comments, but it's looking good.

**Comment by jpkrohling** [comment]
The DCO check is failing. Could you please sign-off your commit? Instructions are here: https://github.com/jaegertracing/jaeger-operator/runs/1192444215

File: deploy/examples/ingress-with-annotations.yaml (hunk 3 above)
Comment (prageethw) [reply deploy/examples/ingress-with-annotations.yaml:15]: I was thinking to show how to use ingress annotations here

File: deploy/examples/ingress-with-annotations.yaml (hunk 4 above)
Comment (prageethw) [reply deploy/examples/ingress-with-annotations.yaml:18]: agent removed

File: deploy/examples/ingress-with-annotations.yaml (hunk 3 above)
Comment (jpkrohling) [reply deploy/examples/ingress-with-annotations.yaml]: Then we can have two examples, each with one use-case? Unless all the people looking for an example of how to set the annotation will also need to set the host, and vice-versa.

**Comment by prageethw** [comment]
closing and will make separate pull with 2 examples

**Pull request closed without merging**